#
# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
#
# Copyright (c) 2010-2017 Oracle and/or its affiliates. All rights reserved.
#
# The contents of this file are subject to the terms of either the GNU
# General Public License Version 2 only ("GPL") or the Common Development
# and Distribution License("CDDL") (collectively, the "License").  You
# may not use this file except in compliance with the License.  You can
# obtain a copy of the License at
# https://oss.oracle.com/licenses/CDDL+GPL-1.1
# or LICENSE.txt.  See the License for the specific
# language governing permissions and limitations under the License.
#
# When distributing the software, include this License Header Notice in each
# file and include the License file at LICENSE.txt.
#
# GPL Classpath Exception:
# Oracle designates this particular file as subject to the "Classpath"
# exception as provided by Oracle in the GPL Version 2 section of the License
# file that accompanied this code.
#
# Modifications:
# If applicable, add the following below the License Header, with the fields
# enclosed by brackets [] replaced by your own identifying information:
# "Portions Copyright [year] [name of copyright owner]"
#
# Contributor(s):
# If you wish your version of this file to be governed by only the CDDL or
# only the GPL Version 2, indicate your decision by adding "[Contributor]
# elects to include this software in this distribution under the [CDDL or GPL
# Version 2] license."  If you don't indicate a single choice of license, a
# recipient has the option to distribute your version of this file under
# either the CDDL, the GPL Version 2 or to extend the choice of license to
# its licensees as provided above.  However, if you add GPL Version 2 code
# and therefore, elected the GPL Version 2 license, then the option applies
# only if the new code is made subject to such option by the copyright
# holder.
#


#connection failures
remote.connect.interrupt=Befehlsausf\u00FChrung unterbrochen f\u00FCr Befehl {0}
remote.connect.ioexception= Befehl {0} konnte nicht \u00FCber SSH ausgef\u00FChrt werden
internal.error=Interner Fehler: {0}
remote.command.summary=Dieser Remote-Befehl wurde ausgef\u00FChrt:\n{0}\nFolgende Ausgabe wurde empfangen:\n{1}
remote.command.error=Windows-Exception mit dieser Meldung ({0}) wurde abgerufen, w\u00E4hrend versucht wurde, diesen Befehl auszuf\u00FChren: {1}
cant.delete=Tempor\u00E4re Remote-Datei konnte nicht mit dem Authentifizierungstoken gel\u00F6scht werden. L\u00F6schen Sie sie manuell.\nHost: {0}  Dateipfad: {1}
